src/payment/strategies/braintree/braintree-hosted-form.spec.ts

```diff
@@ -219,18 +219,38 @@ describe('BraintreeHostedForm', () => {
             .toHaveBeenCalledWith({ fieldType: 'cardCode' });
     });
 
-    it('notifies when card number changes', async () => {
-        const handleCardTypeChange = jest.fn();
+    describe('#cardTypeChange', () => {
+        let handleCardTypeChange: jest.Mock;
 
-        await subject.initialize({
-            ...formOptions,
-            onCardTypeChange: handleCardTypeChange,
+        beforeEach(async () => {
+            handleCardTypeChange = jest.fn();
+
+            await subject.initialize({
+                ...formOptions,
+                onCardTypeChange: handleCardTypeChange,
+            });
         });
 
-        cardFieldsEventEmitter.emit('cardTypeChange', { cards: [{ type: 'Visa' }] });
+        it('notifies when card number changes', () => {
+            cardFieldsEventEmitter.emit('cardTypeChange', { cards: [{ type: 'Visa' }] });
+
+            expect(handleCardTypeChange)
+                .toHaveBeenCalledWith({ cardType: 'Visa' });
+        });
 
-        expect(handleCardTypeChange)
-            .toHaveBeenCalledWith({ cardType: 'Visa' });
+        it('notifies when card number changes and type is master-card', () => {
+            cardFieldsEventEmitter.emit('cardTypeChange', { cards: [{ type: 'master-card' }] });
+
+            expect(handleCardTypeChange)
+                .toHaveBeenCalledWith({ cardType: 'mastercard' });
+        });
+
+        it('notifies when card number changes and type of card is not yet known', () => {
+            cardFieldsEventEmitter.emit('cardTypeChange', { cards: [{ type: 'Visa' }, { type: 'Mastercard' }] });
```

Comment:
Same as above `Visa` and `Mastercard` are not card.type values, it looks like they are lowercase:

<img width="851" alt="image" src="https://user-images.githubusercontent.com/4542735/124250538-51c6d900-db68-11eb-9f54-72ec4c3f5e80.png">